src/leaderboard/messages/momento_request.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{Leaderboard, MomentoResult};
+
+/// A trait that allows Momento request types to define their interaction with the gRPC client.
+pub trait MomentoRequest {
+    #[allow(missing_docs)]
+    type Response;
```

Comment:
Why should this allow missing docs? Is there nothing to say about this, but a reason to not hide it from docs?

Reply:
Copied over from the cache `MomentoRequest`. I've clarified in [2e155bd](https://github.com/momentohq/client-sdk-rust/pull/438/commits/2e155bd21c33ff9a887f04865841f73664ce968b).